modules/assim.sequential/R/Adjustment.R

```diff
@@ -45,8 +45,8 @@ adj.ens<-function(Pf, X, mu.f, mu.a, Pa){
   }
   
   
-  if(sum(mu.a - colMeans(X_a)) > 1 | sum(mu.a - colMeans(X_a)) < -1) logger.warn('Problem with ensemble adjustment (1)')
-  if(sum(diag(Pa) - diag(cov(X_a))) > 5 | sum(diag(Pa) - diag(cov(X_a))) < -5) logger.warn('Problem with ensemble adjustment (2)')
+  if(sum(mu.a - colMeans(X_a)) > 1 | sum(mu.a - colMeans(X_a)) < -1) PEcAn.logger::logger.warn('Problem with ensemble adjustment (1)')
+  if(sum(diag(Pa) - diag(stats::cov(X_a))) > 5 | sum(diag(Pa) - diag(stats::cov(X_a))) < -5) PEcAn.logger::logger.warn('Problem with ensemble adjustment (2)')
```

Comment:
Here's a minor cleanup that I'm suggesting as much to jog a fresh CI run as for any other reason 😼 

```suggestion
  if (sum(mu.a - colMeans(X_a)) > 1 
      || sum(mu.a - colMeans(X_a)) < -1) {
    PEcAn.logger::logger.warn('Problem with ensemble adjustment (1)')
  }
  if (sum(diag(Pa) - diag(stats::cov(X_a))) > 5 
      || sum(diag(Pa) - diag(stats::cov(X_a))) < -5) {
    PEcAn.logger::logger.warn('Problem with ensemble adjustment (2)')
    
  }
```